src/library/Uno.Material/Styles/Controls/TextBox.xaml

```diff
@@ -314,22 +314,22 @@
 									<Storyboard>
 										<DoubleAnimation Storyboard.TargetName="PlaceholderElement_CompositeTransform"
 														 Storyboard.TargetProperty="TranslateY"
-														 Duration="0:0:0.2"
+														 Duration="{StaticResource Material_AnimationDuration}"
```

Comment:
For all translate and scale animations in textbox and passwordbox we can use a specific variable I think it will be clearer.

MaterialTextBoxAnimationDuration ?